examples/AN00156_i2c_master_example/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.21)
+include($ENV{XMOS_CMAKE_PATH}/xcommon.cmake)
+project(AN00156_i2c_master_example)
+
+option(XCOREAI "Build for xcore.ai instead of xcore-200" ON)
```

Comment:
What does this do?

Reply:
Oh, that should be deleted. I'll do it

Reply:
It's in a few CMakeLists.txt, I didn't comment on all of them, sorry.